charts/mission-control/templates/mission-control.yaml

```diff
@@ -388,6 +395,7 @@ spec:
                   {'name': 'Duration', 'text': r.duration_millis != null ? string(r.duration_millis) : 'N/A', 'unit': 'ms', 'headline': true},
                   {'name': 'Success Count', 'value': r.success_count, 'headline': true},
                   {'name': 'Error Count', 'value': r.error_count, 'headline': true},
+                  {'name': 'scraper_id', 'text': r.resource_id},
```

Comment:
Can we make this hidden

Reply:
We don't have hidden implemented for properties, raised a issue in canary checker and flanksource-ui